eng/pipelines/docindex.yml

```diff
@@ -2,3 +2,46 @@ trigger: none
 
 jobs:
   - template: /eng/common/pipelines/templates/jobs/docindex.yml
+
+  - job: UpdateDocsMsBuildConfig
+    pool:
+      vmImage: ubuntu-18.04
+    variables:
+      DocRepoLocation: $(Pipeline.Workspace)/docs
+      DocRepoOwner: MicrosoftDocs
+      DocRepoName: azure-docs-sdk-node
+    steps:
+      # Sync docs repo (this can be sparse)
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+        parameters:
+          SkipDefaultCheckout: true
+          Paths:
+            - ci-configs/packages-latest.json
+            - ci-configs/packages-preview.json
+          Repositories:
+            - Name: $(DocRepoOwner)/$(DocRepoName)
+              # TODO: There is a chicken/egg problem here where we don't know the default branch
```

Comment:
@benbp please correct me if I'm wrong but you can leave the commitish out and it will naturally checkout the default branch.

Reply:
Comments in the script suggest this will use the default branch. Thanks!